### DISCUSSION TITLE
BRANCH: master Error: Input required and not supplied: token

### DISCUSSION THREAD

**Discussion by HaydarJawad** [alshedivat/al-folio#2372] [Q&A] [answered]
Hi all, I wonder if anyone can help. I am getting (BRANCH: master
Error: Input required and not supplied: token) when running   [lighthouse-badger-easy] workflow , when try to run initially. 
The instruction doesn't say anything about a Token  (Recommended Approach). all the best

**Comment by george-gca** [comment, marked as answer]
If you check [.github/workflows/lighthouse-badger.yml](https://github.com/alshedivat/al-folio/blob/492c50964683921299502e3765d36dc227b56436/.github/workflows/lighthouse-badger.yml#L18) there is a `TOKEN_NAME` variable that must be set. You must create a token for your repository in order to be able to use this action. In [lighthouse-badger-action](https://github.com/MyActionWay/lighthouse-badger-action?tab=readme-ov-file#lighthouse-badger-easyyml) there is a step-by-step on how to do this, just go to `lighthouse-badger-easy.yml` -> `Set it up (click to toggle)` -> `2. create a new encrypted repository secret` on the main README file.

**Reply by HaydarJawad** [reply]
Thank you so much it execute perfectly now, but the output is very strange :  not sure why i tried different browsers, i am checking inspect. 

thank you

**Reply by george-gca** [reply]
Check [INSTALL.md Recommended Approach](https://github.com/alshedivat/al-folio/blob/master/INSTALL.md#recommended-approach) step 3.

**Reply by HaydarJawad** [reply]
thank you so much it was "baseurl empty" instruction. You are 100% right. All good now! i will get on with cloning. Thank you again.
